File: backend/app/Models/DocumentoPostulante.php
<?php
namespace App\Models;
use Illuminate\Database\Eloquent\Model;

class DocumentoPostulante extends Model {
    protected $connection = 'mysql_documentos';
    protected $table = 'documento_postulante';
    protected $primaryKey = 'idDocumentoPostulante';
    protected $fillable = [
        'idGradoEst', 'idPostulante', 'codigo_provincia', 'titulo',
        'institucion', 'otraInstitucionn', 'nombreOtraInstitucion',
        'fechaInicio', 'fechaFinaizacion', 'fechaEmision', 'totalHoras'
    ];
    public $timestamps = false;
}

File: Proyecto2-Android/app/src/main/java/com/example/proyecto2_android/models/DocumentoPostulante.kt
package com.example.proyecto2_android.models

data class DocumentoPostulante(
    val idDocumentoPostulante: Int = 0,
    val idGradoEst: Int,
    val idPostulante: Int,
    val codigo_provincia: String,
    val titulo: String,
    val institucion: Int,
    val otraInstitucionn: Int = 0,  // ← Int, no Boolean
    val nombreOtraInstitucion: String? = null,
    val fechaInicio: String,
    val fechaFinaizacion: String,
    val fechaEmision: String,
    val totalHoras: Int
)